pset-iterators.js

```diff
@@ -62,7 +62,15 @@ declared with const
 console.log('Problem 1:')
 
 // Add your code below this line
+const presidentsNamedJames = []
```

Comment:
@kertcodes works as expected, an alternative approach would have been to use `.filter()`